booking-engine-api/use-cases/availability-blocks.md

```diff
@@ -0,0 +1,14 @@
+# Availability blocks
```

Comment:
It's great to have a new page under use cases all about availability blocks, but I need to write new page content.